_includes/feedback.html

```diff
@@ -0,0 +1,23 @@
+<div class="alert alert-info alert-dismissible fade show" role="alert">
+  <p>Please consider providing some quick feedback on how you plan to use MoveIt by clicking the button below. This is
```

Comment:
```suggestion
  <p>Please consider providing some quick feedback on how you plan to use MoveIt to help obtain government funding for further development and prioritize milestones on our roadmap.```

Shorten the sentence